easybuild/framework/easyblock.py

```diff
@@ -1445,7 +1445,20 @@ def extensions_step(self, fetch=False):
 
     def package_step(self):
```

Comment:
@boegel: @stdweird: @fgeorgatos: Agree about all that. 
I don't think I am ready to implement it. Suggestion: you or Fotis can handle the project and I collaborate and learn in the process.